challenges/objects-arrays.js

```diff
@@ -8,27 +8,54 @@
 
 // tyrannosaurus, carnivorous, 7000kg, 12m, Late Cretaceous
 
```

Comment:
Nice Work here! Another cool way would be to make for example -  
class Dino {
  constructor(name, diet, weight, length, period) {
    this.name = name;
    this.diet = diet;
    this.weight = weight;
    this.length = length;
    this.period = period;
  }
}

Reply:
then you could do let rex = new Dino(and list a different type with its list inside)  like ('Tyranosaurus', 'carnivorous',etc)